jest.config.js

```diff
@@ -1,5 +1,13 @@
 module.exports = {
   verbose: true,
   testMatch: ['**/tests/**/*.js?(x)'],
-  testPathIgnorePatterns: ['tests/setup.js']
+  testPathIgnorePatterns: ['tests/setup.js', 'dist'],
+  moduleNameMapper: {
```

Comment:
This fixes some module issues with the tests. It comes from the `react-dnd` documentation about testing.